chalk-ir/src/visit.rs

```diff
@@ -12,12 +12,32 @@ pub mod visitors;
 
 pub use visitors::VisitExt;
 
+// A "result type" that can be returned from a visitor. Visitors pick
```

Comment:
Argh -- text seems good, but these should be "doc comments", meaning they should use `///` so they show up in the rustdoc results

Reply:
done 😅